src/constants.js

```diff
@@ -291,3 +291,22 @@ export const NANO_UPDATE_ADDRESS_LIST_COUNT = 5;
  * Number of elements in the list of transactions of a nano contract
  */
 export const NANO_CONTRACT_HISTORY_COUNT = 5;
+
+/**
+ * Base statuses for saga reducer handlers
+ * Used by all other statuses objects
+ */
+const SAGA_BASE_STATUS = {
```

Comment:
I think you can just call it `BASE_STATUS`

Reply:
Done https://github.com/HathorNetwork/hathor-wallet/pull/695/commits/cedc8b79595c55ce8dcdac5881518afb64b75ae8